indexer/services/comlink/__tests__/lib/restrict-countries.test.ts

```diff
@@ -62,6 +63,11 @@ describe('rejectRestrictedCountries', () => {
 
     rejectRestrictedCountries(req, res, next);
     expect(res.status).toHaveBeenCalledWith(403);
+    expect(res.json).toHaveBeenCalledWith(expect.objectContaining({
```

Comment:
@coderabbitai Is this mock correct?

Reply:
@coderabbitai Where is the `json` and `status` method mocked on the `response` object?